.github/workflows/get-diagnostics.yml

```diff
@@ -17,10 +17,10 @@ jobs:
           ref: ${{ github.head_ref }}
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
you might want to use the `matrix` feature here and test 16. It's going to be a while until we can use node 18 at LI.